client-guide/agile-workflows.html.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Agile workflows
+---
+
+The single biggest problem we have seen in development teams is the lack of an established
+philosophy and process for managing the day-to-day work of the developers and ensuring they are
+delivering value to the business without requiring a constant back and forth. Bureaucracy is
+sacrificed in the name of speed, but this leads to teams that only perform at a fraction of their
+potential, often leaving stakeholders unhappy.
+
+We also see a lot of teams designing their own informal processes, not stopping to consider more
+established Agile methodologies because they are often deemed too complicated or "heavy". Workflows
+such as Scrum and Kanban, however, have been battle-tested by hundreds of thousands of development
+teams all over the world. They have stuck around because _they work_, both for the developers and
+the business, and expecting your own team to do better is unrealistic.
```

Comment:
- I would inject the concept that adapting the methodology to the company is fine, but should be done by keeping the principles of the methodology in mind, rather than cherry-picking individual practices that might even do more bad than good by themselves
- I would also slightly object to appealing to the fact that they're proven to work because they stuck around (e.g. waterfall has stuck around for very long time) but call out that they delivered team and biz success time and time again